Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -1168,6 +1174,11 @@ static void webkitWebViewGetProperty(GObject* object, guint propId, GValue* valu
     case PROP_DEFAULT_CONTENT_SECURITY_POLICY:
         g_value_set_string(value, webkit_web_view_get_default_content_security_policy(webView));
         break;
+    case PROP_THEME_COLOR:
+        WebKitColor* color = new WebKitColor;
```

Comment:
We need to be consistent in the memory allocator being used.

Looking at `Source/WebKit/UIProcess/API/wpe/WebKitColor.cpp` it turns out that the `webkit_color_free()` function calls `WTF::fastFree()`, so this will allocate using the allocator used by the `new` operator, which usually won't be the same used by `WTF::fastFree` when the `GValue` that takes ownership of the boxed `WebKitColor` value destroys it.

So what we want here is `WTF::fastMalloc()`. It would be nicer to have a `webkit_color_new()` helper function, but I think we rather leave that for a separate patch, instead let's do this:

```suggestion
        auto* color = static_cast<WebKitColor*>(fastMalloc(sizeof(WebKitColor)));
```